platforms/ide/problems-api/src/main/java/org/gradle/api/problems/internal/NoOpProblemSummarizer.java

```diff
@@ -14,22 +14,16 @@
  * limitations under the License.
  */
 
-package org.gradle.tooling.internal.protocol;
+package org.gradle.api.problems.internal;
 
-import org.gradle.api.NonNullApi;
-import org.gradle.tooling.internal.protocol.problem.InternalDocumentationLink;
-import org.gradle.tooling.internal.protocol.problem.InternalSeverity;
+import org.gradle.internal.operations.OperationIdentifier;
 
 import javax.annotation.Nullable;
-import java.util.List;
 
-@NonNullApi
-public interface InternalProblemAggregationVersion2 extends InternalProblemAggregation {
```

Comment:
❌ This seems wrong. Removing protocol interfaces can potentially break cross-version compatibility.

Reply:
I think I did this before talking about the TAPI in depth.